style/components/card.css

```diff
@@ -0,0 +1,20 @@
+.card-container {
```

Comment:
This could be `card__container` as the main component is `card` (even though it is outside of card).